lib/src/widgets/adw/header_bar.dart

```diff
@@ -8,9 +8,38 @@ import 'package:gsettings/gsettings.dart';
 import 'package:libadwaita/libadwaita.dart';
 
 class AdwHeaderBar extends StatefulWidget {
-  /// To be used with window_decorations package
+  /// If you use with window_decorations
   /// If you don't want that. use AdwHeaderBar.minimal*
```

Comment:
`AdwHeaderBar.minimal` does exist anymore. Do you mean `custom`?

Overall I think this class needs a better documentation to describe all the different constructors. Since each of them seems to be associated with a given dependency and those dependencies are not listed in the `pubspec.yml`. The user right now, doesn't now those packages (bitsdojo and nativeshell) and does not know where to find them. It makes the class quite complicated to understand.

Reply:
Hmm I will look into docs improvement.